draft-ietf-teep-protocol.md

```diff
@@ -1591,12 +1591,14 @@ This section includes some examples with the following assumptions:
       14            # unsigned(20) / token: /
       50            # bytes(16)
          A0A1A2A3A4A5A6A7A8A9AAABACADAEAF
-      01            # unsigned(1) / supported-cipher-suites: /
-      81            # array(1)
-         83         # array(3)
+      01            # unsigned(1) / supported-ciphersuites: /
+      82            # array(2)
+         83         # array(2)
```

Comment:
This line, L1599 and L1674 should be
82 # array(2)

Reply:
Fixed, thanks!